tests/helpers/__init__.py

```diff
@@ -150,3 +156,25 @@ async def admitted_update_task(
         lambda: workflow_update_exists(client, handle.id, id),
     )
     return update_task
+
+
+async def child_started(
```

Comment:
This does not seem related to child. Maybe a `assert_workflow_exists_eventually` helper instead? Also, this does not need to accept the `workflow` (just use `get_workflow_handle` which is untyped), but when changing to `assert_workflow_exists_eventually` you may want this so you can return a handle for the workflow instead of a bool.